src/core/demangle.d

```diff
@@ -2389,21 +2403,24 @@ private template hasPlainMangling(FT) if (is(FT == function))
     static extern(C) void fooC();
     static extern(Windows) void fooW();
     static extern(C++) void fooCPP();
+    static extern(Objective-C) void fooObjC();
```

Comment:
Not sure if this works, but `extern(Objective-C)` only makes sense on classes.

Reply:
I think `extern (Objective-C)` with classes works only if `macOS && 64bit`.
Should I use `version(OSX) { extern(Objective-C) class Foo { ... ` ?

Reply:
Yes, but use `version(D_ObjectiveC)` instead.